ArduPlane/ArduPlane.cpp

```diff
@@ -357,6 +357,16 @@ void Plane::one_second_loop()
         !is_equal(G_Dt, scheduler.get_loop_period_s())) {
         INTERNAL_ERROR(AP_InternalError::error_t::flow_of_control);
     }
+
+    const float loop_rate = AP::scheduler().get_filtered_loop_rate_hz();
+#if HAL_QUADPLANE_ENABLED
+    if (quadplane.available()) {
+        quadplane.attitude_control->set_notch_sample_rate(loop_rate);
```

Comment:
when these notch rates change, will that inject a disturbance in the notch filter? should we only change when disarmed?

Reply:
This is a poor-mans equivalent to the delta_t stuff which cannot be used for notches. Leonard was ok with the behavior and my testing has not shown any anomalous behavior